DateTimeMate.go

```diff
@@ -26,3 +26,35 @@ func convertRelativeDateToActual(from string) string {
 	}
 	return from
 }
+
+// shrinkPeriod convert a period into a brief period
+// only allow one replacement per each period
+// Ex: 1 hour 2 minutes 3 seconds => 1h2m3s
+// FIXME: almost redundant code for plural & singular -- try to fix with strings.NewReplacer
```

Comment:
You could use a slice with each word and its plural, the code would be clearer with a loop

Reply:
I worked on this and the code is better.  Please review my changes when I post the next PR.

Reply:
Sure, feel free to ping me when you will push it

Reply:
@ccoVeille : Here is a new PR: https://github.com/jftuga/DateTimeMate/pull/6